### PR TITLE
Feature/deactivation form enhancements2

### PR DESCRIPTION
**Pull request by fajardoleo**
Added a message that includes a "Skip" button below the "Don't like to share data" reason option that informs the users that they don't have to share any data and can just skip the opt-in.

### CODE REVIEW

File: assets/scss/admin/deactivation-feedback.scss [hunk 1]
```diff
@@ -31,8 +33,9 @@
 			
 			li.reason {
 				margin-bottom: 10px;
-				
-				.reason-input {
+
+
+				.reason-input, .extra-message {
```
Comment (vovafeldman) [inline assets/scss/admin/deactivation-feedback.scss]: @fajardoleo `extra-message` is not descriptive enough. Use `internal-message` instead.

File: includes/i18n.php [hunk 2]
```diff
@@ -186,6 +186,7 @@
 		'placeholder-what-did-you-expect'          => __( "What did you expect?", 'freemius' ),
 		'reason-didnt-work'                        => __( "The plugin didn't work", 'freemius' ),
 		'reason-dont-like-to-share-my-information' => __( "I don't like to share my information with you", 'freemius' ),
+		'dont-have-to-share-any-data'              => __( "You might have missed it, but you don't have to share any data and can just %s the opt-in. %s", 'freemius' ),
```
Comment (vovafeldman) [inline includes/i18n.php]: @fajardoleo the suffix of ` %s` is no needed. There's no reason to enforce the next button to be part of the sentence. Just append the string where you use it.

File: templates/deactivation-feedback-modal.php [hunk 3]
```diff
@@ -20,8 +20,30 @@
 	$reasons_list_items_html = '';
 
 	foreach ( $reasons as $reason ) {
-		$list_item_classes = 'reason' . ( ! empty( $reason['input_type'] ) ? ' has-input' : '' );
-		$reasons_list_items_html .= '<li class="' . $list_item_classes . '" data-input-type="' . $reason['input_type'] . '" data-input-placeholder="' . $reason['input_placeholder'] . '"><label><span><input type="radio" name="selected-reason" value="' . $reason['id'] . '"/></span><span>' . $reason['text'] . '</span></label></li>';
+		$list_item_classes    = 'reason' . ( ! empty( $reason['input_type'] ) ? ' has-input' : '' );
+
+		if ( isset( $reason['extra_message'] ) && ! empty( $reason['extra_message'] ) ) {
```
Comment (vovafeldman) [inline templates/deactivation-feedback-modal.php]: @fajardoleo same here, as in the other places, refactor from "extra message" to "internal message".

**Comment by fajardoleo** [comment]
@vovafeldman All changes have been made and committed.

**Pull request merged**